reference/tools/data-migration/manage-tasks.md

```diff
@@ -42,9 +44,15 @@ VjX8cEeTX+qcvZ3bPaO4h0C80pe/1aU=
 
 ### Task management overview
 
+Enters the interactive mode to interact with DM-master.
```

Comment:
`Enters` is weird. This is not a code comment.

Reply:
@lilin90 The mistake is resolved in #2118. Thanks for pointing it out.